### DISCUSSION TITLE
2.3.0 Release Time

### DISCUSSION THREAD

**Discussion by isZYKerman** [openzfs/zfs#16811] [General]
When will the 2.3.0 stable release for Ubuntu? I have a pool which had gone through a vdev expansion. It works on TrueNAS since TrueNAS  has supported this feature, but it won't work on Ubuntu, since the version of zfsutils-linux on Ubuntu is still 2.2.6. Is there a roadmap or something that I can get to know when the next version will be released?

**Comment by amotin** [comment]
I don't know a time line for 2.3 release, but here is a PR for 2.3-rc4: https://github.com/openzfs/zfs/pull/16794 .

**Reply by isZYKerman** [reply]
😨So there will be a rc4 stage before the stable release? Okay...

**Comment by n0xena** [comment]
well - I'm using 2.3.0-rc3 on 6.12.1 on arch - so you should be able to build a .deb for your system and use your pool this way

**Reply by isZYKerman** [reply]
OK, I'll try it. Thanks!

**Reply by satmandu** [reply]
This is my own 2.3.0-rc3 ppa that I'm using in my homelab on Ubuntu 24.10 with Linux kernel 6.13.0-rc2: https://launchpad.net/~satadru-umich/+archive/ubuntu/zfs-experimental

Caveat emptor, etc, etc.

**Reply by isZYKerman** [reply]
This is very helpful. Thanks!

**Comment by sparks79** [comment]
Could someone advise me on where to place the following enquiry or answer it here and now?
I am interested in trying out the zfs file system on a linux distro.
I have a powerful system with an i9 CPU and 128gb ram, and four 512GB SSD's.
At present I have the 4 ssd's using intel rst raid0 and linux mint 22.1 beta installed.
So the os see's the array as 2tb raid0 and everything purs like a ketten.
Now I just read that open zfs 2.3.0 has been released.
And I was searching for a linux distro that may support open zfs 2.3.0 with four ssd's in raid0 with the o/s on the raid as well.
For me it's all about speed and backups or redundancy don't worry me at all because I keep all my important stuff on a NAS.
